include/nfc/ndef/file.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef NFC_NDEF_FILE_H_
```

Comment:
I think this library should be moved to nfc/t4t/ directory as ndef_file.h (cc_file.h is there as well). Concept of NDEF files exist only within T4T specification

Reply:
Ok. Done